src/components/headers/navigation.js

```diff
@@ -218,6 +218,9 @@ const Navigation = () => {
           Create Extensions
         </a>
       </NavEntry>
+      <NavEntry>
+        <a href="https://hub.quarkiverse.io/">Publish Extensions</a>
```

Comment:
How about `Share`?

Reply:
Yes, that works!

Reply:
Or `Host your extension`

Reply:
I think Share is better, because it also captures things like 'making it appear on code.quarkus and extensions.quarkus'. And to me, Host implies "Self-host". And no one wants that. :)